mirageio.opam.template

```diff
@@ -1,7 +0,0 @@
-pin-depends: [
-  ["dream-pure.dev"    "git+https://github.com/samoht/dream.git#e3c57b830ca7f517f531ac998ab13923f8bd076d"]
-  ["dream.dev"         "git+https://github.com/samoht/dream.git#e3c57b830ca7f517f531ac998ab13923f8bd076d"]
-  ["dream-mirage.dev"  "git+https://github.com/samoht/dream.git#e3c57b830ca7f517f531ac998ab13923f8bd076d"]
-  ["dream-httpaf.dev"  "git+https://github.com/samoht/dream.git#e3c57b830ca7f517f531ac998ab13923f8bd076d"]
-  ["tailwindcss.dev" "https://github.com/tmattio/opam-tailwindcss/archive/3e60fc32bbcf82525999d83ad0f395e16107026b.tar.gz"]
-]
```

Comment:
I guess we still need this to avoid overwrites when `dune` generates the `mirageio.opam` based on the dune-project.

Reply:
You're right, I've readded it now.